examples/stringification/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# separated executable to ensure there is no specializations of StringMaker leaked
+# In opposite to all_features/stringification.cpp this test intended to check all the bundled
+# output rules for all the fundamental types, std containers etc
+# it might overlap with other tests, but better to test twice, than miss some cases
+add_executable(stringification main.cpp)
+target_link_libraries(stringification doctest)
+target_compile_definitions(stringification PRIVATE SIZEOF_VOID_P=${CMAKE_SIZEOF_VOID_P})
```

Comment:
Just calculate `SIZEOF_VOID_P` at runtime?

Reply:
Then instead of more or less cool-looking compile-time strings `POINTER_PREFIX "abadf00d"` I will need string concatenation. Ew.

This won't be a problem if I rebase this branch against the fix-string branch where pointer "abadf00d" is stringified as "0xabadf00d" independently of SIZEOF_VOID_P.